src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+const Big = require('big.js');
```

Comment:
I think you used the wrong package here. See https://www.npmjs.com/package/big-js, it'a `big-js` not `big.js`.

Also, consider using ES6 `import` instead.